### DISCUSSION TITLE
前段时间还能部署在服务器上，最近下包已经404了

### DISCUSSION THREAD

**Discussion by kunyyds666** [1943time/inkdown#90] [Q&A] [answered]
curl -fsSL https://www.inkdown.me/latest | bash,404我还想要部署上服务器

**Comment by 1943time** [comment]
不好意思哦，我个人精力支撑不了这么多版本，后续可能还得开发在线付费版来支撑这个项目维护下去，所以暂不支持部署版了。

**Reply by kunyyds666** [reply]
您好，有原来的包吗，我还想部署一下，现在我确实有共享编辑的需求，但是手上并没有mac，如果以后上线付费版我一定支持

**Reply by 1943time** [reply]
https://assets.inkdown.me/release/0.53.0.zip
解压这个压缩包按照之前的步骤部署就行，不过部署版缺失了很多细节，已经很久没更新了。

**Reply by kunyyds666** [reply]
谢谢，希望能够继续开源部署版，哪怕没用时间进行更新，因为肯定有很多开发者会愿意共同维护如此优秀，轻便的软件，我真的很喜欢

**Reply by 1943time** [reply, marked as answer]
我的习惯是放出来的东西质量至少得过得去，目前的精力无法支撑部署版的维护，所以情愿停止，谢谢认可，以后如果有机会，我会考虑再开放部署版的。
